src/main/java/tutorial/Disc.java

```diff
@@ -10,6 +10,7 @@
         this.size = size;
     }
 
+    //do we need to expose?
     int getSize() {
```

Comment:
Do we need to expose the disc size?  Maybe we could use Comparable / Comparator or provide an isBiggerThan method or some such?